autopush/webpush_server.py

```diff
@@ -348,6 +348,11 @@ def metrics(self):
     def process(self, command):
         raise NotImplementedError()
 
+    def get_month(self, command):
```

Comment:
I think we'll just be passing through the db.current_msg_month so I think this should go away completely? update_rotating_tables will never change it